src/Sylius/Bundle/UiBundle/Resources/assets/sass/_variables.scss

```diff
@@ -0,0 +1 @@
+$sylius: #1abb9c;
```

Comment:
I'm :+1: with peteward. Separate the color palette. I'd even use `$sylius-main-color` to avoid naming variable with a color in its name which will be confusing if the color it self is changed.